docs/src/Tooltip.doc.js

```diff
@@ -1,165 +1,822 @@
+                label="Desktop"
+                name="device"
+                onChange={() => setDevice( "desktop" )}
+                value="desktop"
+              />
+            </Box>
+             <Box paddingY={1}>
+              <RadioButton
+                checked={device === "tablet"}
+                id="tablet"
+                label="Tablet"
+                name="device"
+                onChange={() => setDevice( "tablet" )}
+                value="tablet"
+              />
+            </Box>
+          </Box>
+        </Flex>
+      </Flex>
+    </ScrollableContainer>
+)}`}
+      />
+    </MainSection.Subsection>
+  </MainSection>,
 );
 
 card(
-  <Example
-    name="Accessibility"
+  <MainSection name="Writing">
+    <MainSection.Subsection>
+      <MainSection.Card
+        cardSize="md"
+        type="do"
+        description={`
+- Describe only the element that triggers Tooltip.
+- Focus on the action by beginning with a verb.
+- Use succinct and scannable language.
+- As much as possible, limit Tooltip to a maximum of 60 to 75 characters.
+- Use sentence case while always capitalizing the word “Pin.”
+`}
+      />
+      <MainSection.Card
+        cardSize="md"
+        type="don't"
+        description={`
+- Describe elements other than the one that triggers Tooltip.
+- Use Tooltip to restate text already visible on screen.
+- Use Tooltip to communicate critical information, such as an error, instructions for performing a task, or interaction feedback.
+`}
+      />
+    </MainSection.Subsection>
+  </MainSection>,
+);
+
+card(
+  <MainSection
+    name="Related"
     description={`
-      If more information is needed to describe an IconButton, you can wrap it in a Tooltip
-      in order to reveal more help text on hover.
+[Flyout](/Flyout)
 
-      Screenreaders will pick up on the accessibilityLabel supplied to the child, in this case
-      IconButton, while the Tooltip just provides a more visible on screen description.
+Flyout displays a lightweight task related to the content on screen. One example of Flyout is the board picker, which allows people to choose the board to save a Pin to. While Tooltips are purely text-based, Flyouts offer broader content options, such as [Buttons](/Buttons) and [Images](/Images).
 
-      Be sure to internationalize your \`text\`.
-  `}
-    defaultCode={`
-<Tooltip inline text="Remove image">
-  <IconButton
-    accessibilityLabel="Delete this image from your profile"
-    bgColor="white"
-    icon="trash-can"
-    iconColor="darkGray"
-    onClick={() => { console.log('😱')}}
-  />
-</Tooltip>
-`}
+[ScrollableContainer](/ScrollableContainer)
+
+ScrollableContainer is needed for proper positioning when Tooltip is anchored to an element that is located within a scrolling container. The use of ScrollableContainer ensures Tooltip remains attached to its anchor when scrolling. See the [With scrollableContainer](#With-scrollableContainer) variant to learn more.
```

Comment:
Don't forget to update the link here (for the ScrollableContainer example)

Reply:
leaving as is as capitalization in side nav is managed